metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -336,6 +336,7 @@ def make_flow(
 
     # Attach @kubernetes and @environment decorator to the flow to
     # ensure that the related decorator hooks are invoked.
+    # TODO why is this needed?  Do I need to add secrets deco to Kubernetes as well?
```

Comment:
This is needed so that we have the default (if needed) decorator values initialized and ready to be attached to the yaml spec for argo workflows.